Gemfile

```diff
@@ -2,5 +2,7 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 gemspec
 
+gem 'aruba'
 gem 'cucumber', '< 3.0' if RUBY_VERSION < '2.1'
+gem 'gitlab' # gem 'gitlab', github: 'NARKOZ/gitlab'
 gem 'octokit', '~> 4.9'
```

Comment:
This was there, but I can't explain why because this should be sufficent:
https://github.com/voxpupuli/modulesync/blob/7202e6bfc819a8e7323a03c9391cf45b2b6df9d2/modulesync.gemspec#L27